src/test/common/test_async_compressor.cc

```diff
@@ -1,225 +0,0 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
you need to update `src/test/common/CMakeLists.txt` accodingly.

```
CMake Error at cmake/modules/BuildBoost.cmake:191 (_add_executable):
  Cannot find source file:

    test_async_compressor.cc

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx
Call Stack (most recent call first):
  src/test/common/CMakeLists.txt:190 (add_executable)
```

see https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=huge/7165//consoleFull